test/test_env.py

```diff
@@ -1420,7 +1453,7 @@ def test_nested_reward(self, envclass):
         MockBatchedLockedEnv,
         MockBatchedUnLockedEnv,
         MockSerialEnv,
-        NestedRewardEnv,
+        # NestedCountingEnv,
```

Comment:
this needs https://github.com/pytorch/rl/issues/1268 fixed to be uncommented

Reply:
ditto